perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -1352,4 +1352,29 @@ has comment_count => (
     },
 );
 
+=item cancel_update_alert
+
+Takes a comment id as single argument and adds to the AlertSent
+table so an alert is not sent when FixMyStreet::Script::Alerts::send_updates
+is run
+
+=cut
+
+sub cancel_update_alert {
```

Comment:
In this commit, could the parts of the code that could call this function be changed to call this function? :) Admin/Triage.pm, Buckinghamshire.pm, ArchiveOldEnquiries.pm. (I'd say also UpdatesBase.pm except that has an extra check to only find alerts by the report user, so perhaps not.) There's also a typo in the commit message.

Reply:
Done a bonus fixup that I think accounts for UpdatesBase, but may be overcomplicating things